about.html

```diff
@@ -0,0 +1,106 @@
+          </div>
+          <div class="logo">
+            <a href="index.html"></a>
+            <img class="img-logo" src="./images/riana.png" alt="Logo">
+          </div>
+      </div>
+      <div class="mobile-menu hide" id="mobile-menu-id">
+          <button id="cross">
+            <img src="./images/Icon.png" alt="">
+          </button>
+          <nav class="menu-list">
+              <ul>
+                  <li><a href="index.html">Home</a></li>
+                  <li><a href="about.html">About</a></li>
+                  <li><a href="#services-id">program</a></li>
+                  <li><a href="#team-id">sponsor</a></li>
+                  <li><a href="#team-id">news</a></li>
+              </ul>
+          </nav>
+      </div>
+  </nav>
+  <div class="nav-top">
+      <div class="desktop-nav-list" id="nav-top-list">
+          <ul>
+              <li><a href=""><i class="fab fa-facebook-f"></i></a></li>
+              <li><a href=""><i class="fab fa-twitter"></i></a></li>
+              <li><a href="">English</a></li>
+              <li><a href=""> My Page</a></li>
+          </ul>
+      </div>
+  </div>
+  <div class="desktop-navbar">
+      <div class="logo">
+          <img src="./images/riana.png" alt="logo-image">
+      </div>
+      <div class="desktop-nav-list">
+          <ul>
+            <li><a href="index.html">Home</a></li>
+              <li><a href="about.html">About</a></li>
+              <li><a href="#team-id">program</a></li>
+              <li><a href="#team-id"> Sponsor</a></li>
+              <li><a href="#team-id"> News</a></li>
+          </ul>
+      </div>
+  </div>
+  </nav>
+  <div class="welcome">
+    <h3 class="text">"online live streaming of WORLD EXPO 2030 BUSAN KOREA CONCERT BTS"</h3>
+    <h1 class="text">[LOCATION] Busan Asiad Main Stadium</h1>
+    </h1>
+  </div>
+  <div class="welcome-description">
+    <p class="welcome-text">
+        [LOCATION] Busan Asiad Main Stadium
+    </p>
+    </div>
+    <div class="event-date">
+        <h2 class="fan">don&#39;t miss the show</h2>
+        <p class="fan-p">@ BTS, The world rocking <strong>boys</strong></p>
+      </div>
+  </div>
+</div>
+</section>
+<section class="songs">
+    <div class="songs-heading">
+        <p>Our recent completed Albums</p>
+        <p id="about-song">Take a look at our recently completed Albums</p>
+    </div>
+    <div class="album">
+        <div class="song1">
+            <img class="song-img1" src="./images/img.png" alt="song2-image">
+        </div>
+        <div class="song2">
+            <img class="song-img2" src="./images/img.png" alt="song2-image">
+        </div>
+    </div>
+</section>
+<footer class="footer-about">
+    <div class="footer-logo-about">
+        <img src="./images/riana.png" alt="logo-image">
+    </div>
+    <div class="footer-description-about">
+        <p>Seats are available for fans</p>
+    </div>
+</footer>
+<script src="./script.js"></script>
+</body>
+</html>
```

Comment:
- [ ] I believe that as the previous reviewer requested you to rework on the about sections that did not match the design, you might have accidentally left out some very key sections and now the design has deviated from the original desired design. The sections missing are:

1. The partners section
2. Check the two images for the previous events are not having that redish hue/ theme
3. There is no footer section on mobile view. 

For this reason, I will reiterate what my predecessor asked and request you to ensure that all the sections that are in the design are present in your design before you request for a review.

**These are the desired design:**

**Desktop:**

[Screencast from 26-10-2022 07:17:45 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/198080102-9320deb2-a136-4166-b793-7a07d626dc15.webm)

**Mobile:**

[Screencast from 26-10-2022 07:18:28 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/198080249-f73adf11-57fe-4bcf-8023-bbbc9559f2c6.webm)